Components/Badge/badge.html

```diff
@@ -14,18 +14,69 @@
         <div class="nav-bar p10">
             <div class="nav-left">
                 <i class="fas fa-bars" id="menu"></i>
-                <div class="nav-header">सरणी UI</div>
+                <div class="nav-header fw900">सरणी UI</div>
             </div>
             <div class="nav-right">
                 <a href="#"><i class="fab fa-github"></i></a>
                 <a href="#"><i class="fab fa-twitter"></i></a>
                 <a href="#"><i class="fab fa-linkedin-in"></i></a>
             </div>
         </div>
-        <div class="side-bar">
-            
+        <div class="side-bar pb40">
+            <a href="/Components/Avatar/avatar.html" class="a--reset p20">Avatar</a>
```

Comment:
`class="a--reset` doesn't give clear indication of what this class is doing, will suggest you to use proper naming convention